automations/python/workflows/write_changelog.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+
+app = os.environ["APP"]
+tag_date = os.environ["DATE"]
+release_body = os.environ["RELEASE_BODY"]
+
+file_path = f"documentation/changelogs/{app}/{tag_date}.md"
+
+with open(file_path, "w") as f:
+    f.write(f"# {tag_date}\n\n")
+    f.write(f"""{release_body}\n""")
```

Comment:
Potential change to make this resilient to working directory changes

```suggestion
import os
from pathlib import Path


app = os.environ["APP"]
tag_date = os.environ["DATE"]
release_body = os.environ["RELEASE_BODY"]

file_path = Path(__file__).parents[3] / f"documentation/changelogs/{app}/{tag_date}.md"

with file_path.open("w") as f:
    f.write(f"# {tag_date}\n\n")
    f.write(f"""{release_body}\n""")
```

Reply:
FWIW with Pathlib you don't even need to use `open`:

```py
file_path.write_text(
    """
    # {tag_date}

    {release_body}
    """
)
```

Reply:
Good point 🙂